NEMO/models.py

```diff
@@ -227,6 +227,8 @@ class Tool(models.Model):
 	name = models.CharField(max_length=100, unique=True)
 	parent_tool = models.ForeignKey('Tool', related_name="tool_children_set", null=True, blank=True, help_text='Select a parent tool to allow alternate usage', on_delete=models.CASCADE)
 	visible = models.BooleanField(default=True, help_text="Specifies whether this tool is visible to users.")
+	_description = models.TextField(db_column="description", null=True, blank=True, default='', help_text="HTML syntax could be used")
+	_image = models.ImageField(db_column="image", upload_to=get_tool_image_filename, blank=True, help_text="An image that represent the tool. Maximum width and height are 500px")
```

Comment:
at the top of the file, change `from NEMO.utilities import send_mail, get_task_image_filename` to `from NEMO.utilities import send_mail, get_task_image_filename, get_tool_image_filename`